code/modules/atmospherics/machinery/portable/pipe_scrubber.dm

```diff
@@ -0,0 +1,159 @@
+/obj/machinery/portable_atmospherics/pipe_scrubber
+	name = "pipe scrubber"
+	icon_state = "pipe_scrubber"
+	density = TRUE
+	max_integrity = 250
+	volume = 200
+	///The internal air tank obj of the mech
+	var/obj/machinery/portable_atmospherics/canister/internal_tank
+	///Is the machine on?
+	var/on = FALSE
+	///What direction is the machine pumping to (into scrubber or out to the port)?
+	var/direction = PUMP_IN
+	///the rate the machine will scrub air
+	var/volume_rate = 1000
+	///List of gases that can be scrubbed
+	var/list/scrubbing = list(
```

Comment:
```suggestion
	var/static/list/scrubbing = list(
```

Reply:
not valid, this list is mutated in ui code